packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -0,0 +1,160 @@
+            data-testid="changelog"
+          >
+            <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
+              <Button
+                disabled
+                className="text-white bg-theme-color-water btn-primary small"
+                data-testid="changelogNewReleaseTag"
+              >
+                New release
+              </Button>
+              <ModalClose
+                onClick={(event) => {
+                  if (typeof onRequestClose === 'function') {
+                    onRequestClose(event);
+                  }
+
+                  dismissChangelog();
+                }}
+                data-testid="changelogModalClose"
+              />
+            </header>
+            <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
+              <Image
+                className="object-cover w-44 h-28 rounded-lg"
+                alt="Post Cover image"
+                src={post.image}
+                fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+                loading="lazy"
+                data-testid="changelogImage"
+              />
+              <h3
+                className="mt-2 font-bold text-theme-label-primary typo-title3"
+                data-testid="changelogTitle"
+              >
+                {post.title}
+              </h3>
+              <time
+                className="text-theme-label-quaternary typo-callout"
+                dateTime={post.createdAt}
+                data-testid="changelogDate"
+              >
+                {postDateFormat(post.createdAt)}
+              </time>
+              <div className="mt-2 w text-theme-label-tertiary typo-callout">
+                {post.summary}
+              </div>
+            </section>
+            <footer className="flex gap-3 items-center py-3 px-4 w-full h-16 border-t border-theme-divider-tertiary">
+              <Button
+                className="btn-tertiary"
+                onClick={dismissChangelog}
+                tag="a"
+                href={post.permalink}
+                data-testid="changelogReleaseNotesBtn"
+              >
+                Release notes
+              </Button>
+              {isExtension && (
+                <Button
+                  tag={isFirefoxExtension ? 'a' : undefined}
+                  href={
+                    isFirefoxExtension
+                      ? 'https://r.daily.dev/firefoxupdate'
+                      : undefined
+                  }
+                  className="bg-cabbage-40 btn-primary"
+                  data-testid="changelogExtensionBtn"
+                  onClick={async () => {
+                    await updateExtension();
+                  }}
+                >
+                  Update extension
+                </Button>
+              )}
+            </footer>
+          </div>
+        )
+      }
+      offset={[6 * 16, 2.5 * 16]}
```

Comment:
What is this?
Looks like magic numbers, maybe we can abstract it rather, if it's even needed.

Reply:
> Wow amazing work, got some questions regarding some elements.
> 
> Also is there a way to mimic it in preview deployment?

Honestly locally I would just update the `alerts.lastChangelog` for my user to some previous date (year 2017 or something). Alternatively  some flagsmith flag to always show latest `daily_updates` post but I don't know about that.

Reply:
> What is this? Looks like magic numbers, maybe we can abstract it rather, if it's even needed.

It's just to position the tooltip as per design. I agree with magic numbers point, probably I could just extract it to the top and do something like `const tooltipArrowOffset = [6 * 16, 2.5 * 16]`

Reply:
That last one would work for me :)

Reply:
Done